.github/workflows/test.yml

```
@@ -25,14 +25,21 @@ jobs:
     - name: Install dependencies
       run: npm ci
     
-    - name: Run tests with coverage
-      run: npm run test:cov
-    
+    - name: Run tests with coverage and generate JUnit XML
+      run: JEST_JUNIT_CLASSNAME="{filepath}" JEST_JUNIT_OUTPUT_DIR=./junit JEST_JUNIT_OUTPUT_NAME=junit.xml npm run test:cov -- --reporters=jest-junit
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)